### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by niravpansuriya53**

### CODE REVIEW

File: task3.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+#list of list of string convcert to one list
+flat_list = []
+for sublist in list_of_string:
+    for item in sublist:
+        flat_list.append(item)
+
+#Count the word
+from collections import Counter
+words_to_count = (word for word in flat_list)
```
Comment (himanshu-suthar-tagline) [inline task3.py]: What are you doing here? Would you please explain me why you wrote this line?

File: task3.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+count= Counter(words_to_count)
+cout_word=dict(count.most_common(100000))
```
Comment (himanshu-suthar-tagline) [inline task3.py]: Please explain me why you have used 100000?

File: task3.py (hunk 1 above)
Comment (niravpansuriya53) [reply task3.py]: Create a generator

File: task3.py (hunk 2 above)
Comment (niravpansuriya53) [reply task3.py]: count the max 100000 words

File: task3.py (hunk 1 above)
Comment (himanshu-suthar-tagline) [reply task3.py]: There is no need of creating a generator. Counter will also work if you pass only flat_list to the Counter(flat_list).

File: task3.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+#list of list 
+list_of_list= []
+for element in sentences:
+    sub = element.split(', ')
+    list_of_list.append(sub)    
+print(list_of_list)
+
+#list of list of string
+list_of_string=[]
+for list in list_of_list:
```
Comment (himanshu-suthar-tagline) [inline task3.py]: change `for list in list_of_list` to something meaningful. Also `list` is a reserved keyword, you can't use it in the for loop. Please be careful while using variable name.

**Pull request merged**